_includes/post.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title"    : "{{ post.title }}",
+  "category" : "{{ post.category }}",
+  "tags"     : [{% for tag in post.tags %}"{{ tag }}",{% endfor %} null],
```

Comment:
Nope, it was instead of having a check for unless last to deal with commas. Fixed.